bigbluebutton-html5/imports/ui/Types/meetingClientSettings.ts

```
@@ -834,18 +813,14 @@ export interface VirtualBackgrounds {
 export interface Private {
   analytics: Analytics
   app: App2
-  prometheus: Prometheus
 }
 
-export interface Analytics {
-  includeChat: boolean
-}
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface Analytics {}
```

Biome (2.4.7)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)